packages/plexus/src/zoom/ZoomManager.tsx

```diff
@@ -45,6 +45,8 @@ type TZoomProps = {
   contentHeight: number;
   contentWidth: number;
   viewAll: () => void;
+  zoomAll: () => void;
+  trimAll: () => void;
```

Comment:
why inconsistent naming? Can we cann these zoomIn / zoomOut?

Reply:
Done!